core/vulkan/perfetto_producer/perfetto_threadlocal_emitter.h

```diff
@@ -51,6 +51,9 @@ class ThreadlocalEmitter : ThreadlocalEmitterBase {
       const typename perfetto::DataSourceBase::SetupArgs&) override;
   void StopTracing() override { enabled_ = false; }
   bool Enabled() { return enabled_; }
+  bool CategoryEnabled(const char* category) {
+    return enabled_categories_.contains(category);
```

Comment:
My other PR also makes the assumption if enabled_categories_ is empty, then all categories are enabled.

return enabled_categories.empty() || ...

Reply:
Done.